controllers/admin_controller.js

```diff
@@ -30,20 +31,36 @@ async function handleAddUser(req, res) {
     }
 }
 
-async function handleShowUser(req, res) {
-    const showAllUser = await dbHandler.showAll();
+async function handleShowAllUsers(req, res) {
+    const showAllUser = await User.find({});
     return res.send(JSON.stringify(showAllUser));
 }
 
 async function handleDeleteUser(req, res) {
-    const user = await dbHandler.getUserByEmail(req.body.email);
-    if (user) {
-        await dbHandler.deleteUser(user.email);
-       return  res.status(200).json({ message: "The user has been deleted" })
+    try {
+        await User.findOneAndDelete({'email': req.body.email});
+        return res.status(200).json({message: "The user has been deleted"})
+    }catch (e){
+        return res.status(401).json({message: e.message})
+    }
+}
+
+async function handleShowUser(req, res){
+    const user = await dbHandler.getUserByEmail(req.params.email);
+    return res.send(JSON.stringify(user));
+}
+
+async function handleSaveUpdateAdmin(req, res){
+    try {
+        console.log(req.body)
```

Comment:
- consol.log is redundant here!
- use logger
- do not print debug-logs on production!